app/views/moderator.py

```diff
@@ -33,15 +34,16 @@ class ManageGameView(View):
 
     def get(self, request):
         game = most_recent_game()
+
         participants = Participant.objects.filter(game=game, active=True)
-        humans = list(chain(Spectator.objects.filter(game=game),
-                            Player.objects.exclude(role=PlayerRole.ZOMBIE).filter(game=game, active=True)))
-        zombies = list(chain(Spectator.objects.filter(game=game),
-                             Player.objects.exclude(role=PlayerRole.HUMAN).filter(game=game, active=True)))
+        spectators = Spectator.objects.filter(game=game)
+        humans = Player.objects.filter(game=game, active=True).exclude(role=PlayerRole.ZOMBIE)
```

Comment:
this should be `humans = Player.objects.filter(game=game, active=True, role=PlayerRole.HUMAN)`. Same with zombies = ...

i think?

Reply:
Oh yeah, this was an artifact from the `Player` model